packages/SidePanel/src/components/Overlay/Overlay.styles.js

```diff
@@ -2,11 +2,22 @@ import tokens from "@paprika/tokens/lib/tokens";
 import { css } from "styled-components";
 
 export const overlayCSS = css`
-  background-color: ${tokens.modal.backdrop.backgroundColor};
+  background: ${tokens.modal.backdrop.backgroundColor} none;
+  border: none;
+  border-radius: 0;
   box-sizing: content-box;
   display: block;
   height: 100%;
   position: fixed;
   top: 0;
   width: 100%;
+  
+  &:hover {
+    background: ${tokens.modal.backdrop.backgroundColor} none;
+  }
+  
+  &:active {
```

Comment:
this is extra styling is no longer required, @oscarkwan already changed the <Button isSemantic={false} /> for <RawButton />, the first one introduced weird styles to the overlay, just rebase and remove this style and should be ok.